spec/models/item_spec.rb

```diff
@@ -22,12 +54,19 @@
         expect(@item_1.format_price).to eq("$12.34")
       end
     end
+
     describe "#created_format" do
-      it "returns created date formated as weekday, month day, year" do
+      xit "returns created date formated as weekday, month day, year" do
```

Comment:
Is there a reason to skip this test?

Reply:
This was a test from a previous US that needed to implement a stub. I think Jimmy said he fixed this in the branch for US15 so I think we should be good to go now that I've pulled main into this branch.